modules/mobile/app/controllers/mobile/messaging_controller.rb

```diff
@@ -34,7 +34,7 @@ def use_cache?
     def pagination_params
       {
         page: params[:page],
-        per_page: params[:per_page]
+        per_page: params[:per_page] || 100
```

Comment:
Originally discussed setting this to 1000 to ensure there is never more than 1 page but the pagination code is a generic collections class that is used elsewhere and sets a max of 100.

Reply:
I played with adding in our own pagination helper in place of the generic one but FE provides a `page` parameter (but not a `per_page`) that isn't inline with our Mobile pagination params format so if we did want to make this switch, it'd require FE changes. Since this issue can be easily solved by upping the default per_page on the current pagination params, I don't think it's worth the effort if FE doesn't implement pagination in the app for this feature

Reply:
I'm also hesitant to just rip out the pagination since FE seems to kind of expect it (since they provide `page:1` param). so ripping it out may cause issues.